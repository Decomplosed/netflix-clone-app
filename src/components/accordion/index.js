import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
