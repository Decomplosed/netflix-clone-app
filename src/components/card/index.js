import React from 'react';

export const FeatureContext = createContext();
