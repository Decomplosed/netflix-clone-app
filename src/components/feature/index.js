import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
