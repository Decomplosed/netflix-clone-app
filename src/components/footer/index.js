import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
