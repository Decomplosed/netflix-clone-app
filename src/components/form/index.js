import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
