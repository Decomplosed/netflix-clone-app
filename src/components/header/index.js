import React from 'react';

export default Header({bg = true, children, ...restProps}) {}