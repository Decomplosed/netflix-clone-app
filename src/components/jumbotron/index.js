import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
