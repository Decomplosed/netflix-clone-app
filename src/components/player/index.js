import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
