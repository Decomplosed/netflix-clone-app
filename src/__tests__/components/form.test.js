import React from 'react';
import { render } from '@testing-library/react';
import { Form } from '../../components';

jest.mock('react-router-dom');
