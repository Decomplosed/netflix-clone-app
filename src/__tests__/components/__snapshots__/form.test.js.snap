// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> Renders the <Form /> with an error 1`] = `
<div
  class="form__Container-sc-1uy70tk-0 kBSbhU"
>
  <div
    class="form__Error-sc-1uy70tk-2 gUhRlc"
  >
    Your email address is already being used
  </div>
  <button
    class="form__Submit-sc-1uy70tk-8 kpyGwj"
    type="submit"
  >
    Sign In
  </button>
</div>
`;

exports[`<Form /> Renders the <Form /> with populated data 1`] = `
<div
  class="form__Container-sc-1uy70tk-0 kBSbhU"
>
  <h1
    class="form__Title-sc-1uy70tk-3 kJZlXS"
  >
    Sign In Now
  </h1>
  <form
    class="form__Base-sc-1uy70tk-1 cgNNLe"
  >
    <input
      class="form__Input-sc-1uy70tk-7 FyxXs"
      placeholder="Email address"
    />
    <input
      class="form__Input-sc-1uy70tk-7 FyxXs"
      placeholder="Password"
      type="password"
    />
    <button
      class="form__Submit-sc-1uy70tk-8 kpyGwj"
      disabled=""
      type="submit"
    >
      Sign In
    </button>
  </form>
  <p
    class="form__Text-sc-1uy70tk-4 jwOXwz"
  >
    New to Netflix? 
  </p>
  <p
    class="form__TextSmall-sc-1uy70tk-5 cNJKwz"
  >
    This page is protected by Google reCAPTCHA to ensure you're not a bot. Learn more.
  </p>
</div>
`;
